about.html

```diff
@@ -1,64 +1,125 @@
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
+      <header>
+        <nav>
+          <div class="logo">
+            <img src="/img/lambda-black.png" />
+          </div>
+          <div class="links">
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="#">Contact</a>
+          </div>
+        </nav>
 
-    Let's Work Together
+        <section class="jumbo">
+          <img src="/img/jumbo-about.png" alt="Jumbo image" />
+        </section>
+      </header>
+
+      <section class="top-content">
+        <div class="text-container-about">
+          <h2>About</h2>
+
+          <p>
+            Leverage agile frameworks to provide a robust synopsis for high
+            level overviews. Iterative approaches to corporate strategy foster
+            collaborative thinking to further the overall value proposition.
+            Organically grow the holistic world view of disruptive innovation
+            via workplace diversity and empowerment.
+          </p>
+        </div>
+      </section>
+
+      <section class="middle-content">
+        <div class="box about">
+          <img src="img/about-plan.png" alt="strategy" />
+          <div class="text-container-about-boxes">
+            <h2>Strategy</h2>
+
+            <p>
+              Iterative approaches to corporate strategy foster collaborative
+              thinking to further the overall value proposition.
+            </p>
+          </div>
+
+          <a href="#" class="rounded-button">Learn More</a>
+        </div>
+
+        <div class="box about">
+          <img src="img/about-working.png" alt="strategy" />
+          <div class="text-container-about-boxes">
+            <h2>How We Work</h2>
+
+            <p>
+              Iterative approaches to corporate strategy foster collaborative
+              thinking to further the overall value proposition.
+            </p>
+          </div>
+
+          <a href="#" class="rounded-button">Learn More</a>
+        </div>
+
+        <div class="box about">
```

Comment:
Don't forget to clean up your code. This about class is assigned but never used.